src/services/EventBus.ts

```diff
@@ -11,8 +11,13 @@ type GenericEventHandler = Handler<Events[keyof Events]> | WildcardHandler<Event
 type ExtendedEmitter = Emitter<Events> & {
 	once<Key extends keyof Events>(type: Key, handler: Handler<Events[Key]>): void
 	once(type: '*', handler: WildcardHandler<Events>): void
+	cancelOnce<Key extends keyof Events>(type: Key, handler?: Handler<Events[Key]>): void
+	cancelOnce(type: '*', handler: WildcardHandler<Events>): void
+	_onceHandlersMap: Map<keyof Events | '*', Map<GenericEventHandler, GenericEventHandler>>
```

Comment:
Nitpick: We don't need it inside the `EventBus` object as a public interface. It can be just a variable in the module scope.

Reply:
To not duplicate type in a name. Like we don't add "Array" or "String" prefixes to other variables.

```suggestion
	_onceHandlers: Map<keyof Events | '*', Map<GenericEventHandler, GenericEventHandler>>
```